README.md

```diff
@@ -0,0 +1,146 @@
+# Status endpoint
+Dit document is een richtlijn naar inrichting van de health check endpoint ingebouwd op alle REST API's. 
+Alle endpoints komen onder de resource `/status` terecht.
+
+## Health
+Deze endpoint beschrijft de werking van de service. Als deze niet werkende is zullen onderstaande niveaus worden gebruikt.
+
+#### Status niveau's
+Er zijn 3 verschillende niveaus van health status gedefinieerd.
+
+status|verklaring
+-------|-------
+ok|de service werkt 
+degraded|de service functioneert niet meer op 100% niet kritieke systemen zijn niet bereikbaar of de applicatie verwerkt trager requests
```

Comment:
de service functioneert niet meer op 100%, niet kritische systemen zijn niet bereikbaar of de applicatie verwerkt requests trager